includes/classes/class-content.php

```diff
@@ -10,6 +10,7 @@ public function __construct() {
 		add_filter( 'the_title', [ $this, 'content' ] );
 		add_filter( 'wp_title', [ $this, 'content' ] );
 		add_filter( 'document_title', [ $this, 'content' ] );
+		add_filter( 'pre_get_document_title', [ $this, 'add_city_to_the_page_title' ], 99 );
```

Comment:
You actually introduce a new bug for any SEO plugins for WordPress.

The solution should be: `add_filter( 'pre_get_document_title', [ $this, 'content' ], 99 );` and please remove this:
```php
public function add_city_to_the_page_title() {
	return get_the_title();
}
```